d3a_api_client/redis_client_base.py

```diff
@@ -313,4 +307,16 @@ def on_trade(self, trade_info):
         pass
 
     def on_finish(self, finish_info):
-        pass
\ No newline at end of file
+        pass
+
+    def add_to_batch(self, ):
```

Comment:
I would have to name this too , I am not sure if this makes sense `device.add_to_batch.offer()`

Reply:
I think this is ok, but how about this: 
`device.add_to_batch_commands.offer()`

Reply:
@Hassan754 i dont see this function in the code diff. did you change this?

Reply:
ah, ok, you changed somethin later . OK

Reply:
Works for me, done.